controller/app.controller.js

```diff
@@ -0,0 +1,16 @@
+const {selectTopics} = require("../model/app.model.js")
+
+
+const getAllTopics = (req, res, next) => {
+    selectTopics().then((data) => {
+        res.status(200).send({status: 200, rows: data})
```

Comment:
We don't need to do {status: 200} in our send object, we just want to send the data back, as we are already sending the status by using res.status(200)

You could refactor this like so...
```
selectTopics().then((data) => {
        res.status(200).send({rows: data})
```
        
        You could also refactor it further by sending it like 
   ```
     .then((data) => {
        res.status(200).send({data})
```
        
        In which case, data will be on the key of data